File: Aula08-Exercício/Computador.py
from abc import ABC, abstractmethod

class Computador(ABC):
    def __init__(self,modelo, cor, preco):
        self.modelo = modelo
        self.cor = cor
        self.preco = preco

    @abstractmethod
    def cadastrar(self):
        pass

    def getInformacoes(self):
        texto = f"Modelo: {self.modelo}, Cor: {self.cor}, Preço: {self.preco}"
        return texto

    

File: Aula09-ExercícioEmJava/Computador.java
public abstract class Computador{

    public String modelo, cor;
    public double preco;

    public Computador(){

    }

    public Computador(String modelo, String cor, double preco){
        this.modelo = modelo;
        this.cor = cor;
        this.preco = preco;
        //System.out.println(this.modelo + " construído!");
    }

    public abstract void cadastrar();

    public String getInformacoes(){
        return  "Modelo: " + this.modelo + "\n" + 
                "Cor: " + this.cor + "\n" +
                "Preço: R$ " + this.preco;
    }

    @Override
    public String toString() {
        return getInformacoes();
    }

}